app/src/main/res/layout-land/ongoing_story_card.xml

```diff
@@ -92,3 +87,4 @@
     </androidx.constraintlayout.widget.ConstraintLayout>
   </com.google.android.material.card.MaterialCardView>
 </layout>
+
```

Comment:
Revert this.

Reply:
@aggarwalpulkit596  Done. Please review the changes made